client/a8c-for-agencies/sections/marketplace/products-overview/multi-product-card/index.tsx

```diff
@@ -197,6 +197,7 @@ export default function MultiProductCard( props: Props ) {
 							<Button
 								className="product-card__select-button"
 								variant={ isSelected ? 'secondary' : 'primary' }
+								tabIndex={ -1 }
```

Comment:
We don't need to make this button tabbable since the card itself already acts as the button.